packages/flutter/test/material/raw_material_button_test.dart

```diff
@@ -11,97 +11,356 @@ import 'package:flutter_test/flutter_test.dart';
 
```

Comment:
`raw_material_button_test`

Before:
<img width="500" alt="rmb_before" src="https://user-images.githubusercontent.com/7915388/103942445-3dd47a00-50e5-11eb-98c4-6680fba30199.png">

<img width="500" alt="rmb_after" src="https://user-images.githubusercontent.com/7915388/103942449-3f9e3d80-50e5-11eb-8a67-69a5e2d94b5a.png">

Reply:
Why would we need tests for both InkRipple and InkSplash?